src/Fixer/Operator/NewWithParenthesesFixer.php

```diff
@@ -210,6 +210,13 @@ private function ensureNoParenthesesAt(Tokens $tokens, int $index): void
             return;
         }
 
+        // Check if there's an object operator after the closing parenthesis
+        // Preserve parentheses in expressions like "new A()->method()" as per RFC
+        $afterClosingIndex = $tokens->getNextMeaningfulToken($closingIndex);
+        if (null !== $afterClosingIndex && $tokens[$afterClosingIndex]->isObjectOperator()) {
```

Comment:
Is the part `null !== $afterClosingIndex` needed? There always is a token after `)`, right?